src/events.rs

```diff
@@ -122,6 +122,31 @@ macro_rules! make_mouse_event {
     };
 }
 
+macro_rules! make_pointer_event {
+    ($name:ident, $type:literal) => {
+        make_mouse_event!($name, $type => web_sys::PointerEvent);
+
+        impl $name {
+            #[inline] pub fn pointer_id(&self) -> i32 { self.event.pointer_id() }
+
+            #[inline] pub fn width(&self) -> i32 { self.event.width() }
+            #[inline] pub fn height(&self) -> i32 { self.event.height() }
+
+            #[inline] pub fn pressure(&self) -> f32 { self.event.pressure() }
+            #[inline] pub fn tangential_pressure(&self) -> f32 { self.event.tangential_pressure() }
+
+            #[inline] pub fn tilt_x(&self) -> i32 { self.event.tilt_x() }
+            #[inline] pub fn tilt_y(&self) -> i32 { self.event.tilt_y() }
+
+            #[inline] pub fn twist(&self) -> i32 { self.event.twist() }
+
+            #[inline] pub fn pointer_type(&self) -> String { self.event.pointer_type() }
```

Comment:
This should probably return an enum. Since web-sys doesn't return an enum, maybe we should leave this out for now.